### PR TITLE
Update the HTML to SSML function to use child text items instead of using the textContent attribute'

### PR DESCRIPTION
**Pull request by jacksonh**
- Add a new HTML to SSML function
- Use the new HTML -> SSML function

### CODE REVIEW

File: packages/text-to-speech/src/htmlToSsml.ts [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+export type VoiceOptions = {
+  primary: string
+  secondary: string
+}
+
+const startSsml = (element: Element, voices: VoiceOptions): string => {
+  const voice = element.nodeName === 'BLOCKQUOTE' ? voices.secondary : voices.primary
+  return `
+<speak xmlns="http://www.w3.org/2001/10/synthesis" xmlns:mstts="http://www.w3.org/2001/mstts" xmlns:emo="http://www.w3.org/2009/10/emotionml" version="1.0" xml:lang="en-US"><voice name="${voice}"><prosody rate="0%" pitch="0%">
```
Comment (sywhb) [inline packages/text-to-speech/src/htmlToSsml.ts:143]: Maybe we should put rate=`1` or `default` according to the doc:
```Indicates the speaking rate of the text. You can express rate as:
A relative value, expressed as a number that acts as a multiplier of the default. For example, a value of 1 results in no change in the rate. A value of 0.5 results in a halving of the rate. A value of 3 results in a tripling of the rate.
A constant value:
x-slow
slow
medium
fast
x-fast
default```

Comment (sywhb) [reply packages/text-to-speech/src/htmlToSsml.ts:143]: Maybe we should put pitch=`default` and some value:
```Indicates the baseline pitch for the text. You can express the pitch as:
An absolute value, expressed as a number followed by "Hz" (Hertz). For example, <prosody pitch="600Hz">some text</prosody>.
A relative value, expressed as a number preceded by "+" or "-" and followed by "Hz" or "st" that specifies an amount to change the pitch. For example: <prosody pitch="+80Hz">some text</prosody> or <prosody pitch="-2st">some text</prosody>. The "st" indicates the change unit is semitone, which is half of a tone (a half step) on the standard diatonic scale.
A constant value:
x-low
low
medium
high
x-high
default```

Comment (sywhb) [reply packages/text-to-speech/src/htmlToSsml.ts:143]: We should maybe add language attribute to the `speak` element too

Comment (sywhb) [reply packages/text-to-speech/src/htmlToSsml.ts:143]: `xml:lang="en-US"`

Comment (jacksonh) [reply packages/text-to-speech/src/htmlToSsml.ts:143]: Ah thanks, I will copy over the values you had

Comment (jacksonh) [reply packages/text-to-speech/src/htmlToSsml.ts:143]: I think i C&P'd this header from Azure demo app

Comment (jacksonh) [reply packages/text-to-speech/src/htmlToSsml.ts]: ok updated these ones to pass in from config. I left pitch as hard coded.

**Pull request merged**